docs/interaction design/meeting prep.tex

```diff
@@ -0,0 +1,105 @@
+4285034}
+\affil[ ]{Health Informatics Group C}
+
+\date{\today}
+
+\renewcommand*\contentsname{Summary}
+
+\begin{document}
+\maketitle
+
+\section{Relevant}
+the Final Report will also
+contain a section describing the HCI module that was realized for the user interaction with
+the developed solution. This section will reveal what the students learned in the Interaction
+Design course and will be evaluated by the corresponding lecturer. The grade for Interaction
+Design will be assigned based on the content of this section (see Section 8 for assessment
+process and criteria). $\to$ Dit mag dus ongeveer 2 a4'tjes omvatten; zie: \ref{sec:Referentie}:\ref{itm:ite}
+
+De volgende punten zijn interessant voor het $ID$ deel van het final report: 
+\begin{enumerate}
+\item User Centered Design: \begin{itemize}
+	\item vroeg in het proces naar gebruiker en context gekeken
+	\item evaluatie met wenxin gehad
+	\item iteratief ontwerp
+	\item mono-disciplinair team
+	\end{itemize}
+\item reden voor user involvement is voornamelijk vanuit pragmatisch en commitment rationale
+\item contextual inquiry  \begin{itemize}
+	\item requirements zijn voor ons opgesteld door Willem-Paul
+	\item gesprekken met wenxin ivm haar werk
+	\end{itemize}
+\item cultural probe : niet toegepast
+\item persona : beschrijving van Wenxin Wang
+\item We maken gebruik van scenario's in de vorm van \begin{itemize}
+	\item conceptuele scenario's (uitwerkvoorbeelden)
+	\item use cases (sprint plans)
+	\item concrete scenario's (demo met wenxin)
+	\item gebruikersverhalen (ongebruikt)
+	\end{itemize}
+\item high-fidelity prototyping 
+\item empirical evaluations (demo with wenxin) through debriefing
+\item predictive evaluation (prototype)
+\item geen rekening gehouden met perceptuele aspecten en motoriek (irrelevant voor ons project)
+\item geheugen is relevant bij bijv. het werken met de taal. Hiermee houden we rekening dmv de language manual
+\item hoe hebben wij in het programma rekening gehouden met menselijk falen?
+\item houden geen rekening met ontwerp voor kinderen.
+\item houden ook geen rekening met kleurenblindheid omdat de doelgroep zeer specifiek is.
+\item we houden geen rekening met emotionele toestanden omdat het een professioneel programma is
+\item We houden wel rekening met flow, door zo veel mogelijk handelingen te automatiseren
+\item antropomorfisme of collaboratie, social awareness en persuasive technology n.v.t.
+\end{enumerate}
+
+\section{Beschrijving Final Report} \label{sec:Referentie}
+This deliverable is the main document about the developed, implemented, and validated
+software product. It will present the main functionalities of the product and discuss to which
+extent they satisfy the needs of the user. For this purpose, an evaluation of the functionalities
+performed using a well-justified method needs to be presented, as well as a failure analysis ?
+where the product does not perform as needed. Furthermore, the Final Report will also
```

Comment:
Dit is letterlijk gekopieerd uit de guidelines.pdf om te refereren naar beschrijving van het HCI deel in het final report